catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -414,12 +405,10 @@ private static void parseTopLevel(RegistryObjectType registryObject, MetacardImp
 
         if (registryObject.isSetExternalIdentifier()) {
             List<ExternalIdentifierType> extIds = registryObject.getExternalIdentifier();
-            for (ExternalIdentifierType extId : extIds) {
-                if (extId.getId()
-                        .equals(RegistryConstants.REGISTRY_MCARD_LOCAL_ID)) {
-                    metacard.setId(extId.getValue());
-                }
-            }
+            extIds.stream()
+                    .filter(extId -> extId.getId()
+                            .equals(RegistryConstants.REGISTRY_MCARD_LOCAL_ID))
+                    .forEach(extId -> metacard.setId(extId.getValue()));
```

Comment:
With this and the comment on the other PR, I decided to look into stream efficiency and it actually looks a bit worse than normal loops. The trade off is (potentially) cleaner code which may be safer.   :|

Will look into findFirst for here though.